src/main/java/org/gridsuite/study/server/service/SupervisionService.java

```diff
@@ -90,6 +90,7 @@ public Integer deleteComputationResults(ComputationType computationType, boolean
             case SENSITIVITY_ANALYSIS:
                 return dryRun ? sensitivityAnalysisService.getSensitivityAnalysisResultsCount() : deleteSensitivityAnalysisResults();
             case SHORT_CIRCUIT:
+            case SHORT_CIRCUIT_ONE_BUS:
```

Comment:
admin-tools is currently passing SHORT_CIRCUIT to remove all results from short-circuit repository (both all-buses and 1-bus results). In deed this repository cannot make the difference.
So I think it's useless to add SHORT_CIRCUIT_ONE_BUS here cause it wont be passed by admin-tool.

Reply:
We could manage both separately and split `getShortCircuitResultsCount` and `deleteShortcircuitResults` methods or consider that giving SHORT_CIRCUIT or SHORT_CIRCUIT_ONE_BUS it delete both results.
Maybe we could add a TODO, but it doesn't change anything here, the same admin-tools script which call with SHORT_CIRCUIT will still get both results count and delete both results type